apps/11gist-blog/script.js

```diff
@@ -1,5 +1,71 @@
 'use strict';
 
-$(document).ready(function() {
+$(document).ready(function () {
   // You code here
+  $.ajax('https://api.github.com/users/UnimportantWank/gists', {
+    success: function (description) {
+      console.log(description);
+      // fliltering out anything that isn't a #post
+      let filtered = description.filter(function (each) {
+        return each.description.split(' ').includes("#post");
+      })
+      // getting rid of the posts text
+      function filterPosts(text) {
+        return text.split('#post').join(' ').trim();
+      }
+      //log the filtered posts
+      console.log(filtered);
+      filtered.forEach(function (each) {
+        console.log(each);
+        var string = `
```

Comment:
Nice template string here. This should be sent through the marked library though